### DISCUSSION TITLE
Should `authorized_scope` be exposed as a Rails helper method?

### DISCUSSION THREAD

**Discussion by shepmaster** [palkan/action_policy#205] [Ideas]
`allowed_to?` is exposed as a helper method for views: 

https://github.com/palkan/action_policy/blob/ad92f0761f4730bf699ae40b03f06cf88fe8746b/lib/action_policy/rails/controller.rb#L26

Is there a strong reason that `authorized_scope` should not also be exposed? 

My specific context is a select menu that wants to list the related models that the current user can see.

**Comment by palkan** [comment]
Good question.
Yeah, I think, keeping in mind your use case, that we should do that.
Added: https://github.com/palkan/action_policy/commit/d3acc6c4c7929cea2f5917b491bfa9a351338558
